### PR TITLE
consider scroll max iteration and ScrollDelta from capabilities 

### PR DESCRIPTION
**Pull request by rahul-kumar-sdet**
Co-authored By: purniram [purnima.ramkumar@thoughtworks.com](mailto:purnima.ramkumar@thoughtworks.com)
Co-authored By: dharsanasridharan [dharsana.sridharan@thoughtworks.com](mailto:dharsana.sridharan@thoughtworks.com)

### CODE REVIEW

**Comment by sudharsan-selvaraj** [comment]
@rahul-kumar-sdet Thank you for submitting the PR. Could you also update the server version to `0.0.25` in the files listed below?

https://github.com/AppiumTestDistribution/appium-flutter-server/blob/main/server/pubspec.yaml#L3
https://github.com/AppiumTestDistribution/appium-flutter-server/blob/main/server/lib/src/runner.dart#L13

File: server/lib/src/utils/element_helper.dart [hunk 1]
```diff
@@ -492,8 +486,10 @@ class ElementHelper {
     Duration? settleBetweenScrollsTimeout,
     Duration? dragDuration,
   }) async {
-    delta ??= defaultScrollDelta;
-    maxScrolls ??= defaultScrollMaxIteration;
+    delta ??= FlutterDriver.instance.settings
+        .getSetting('flutterScrollDelta');;
```
Comment (sudharsan-selvaraj) [inline server/lib/src/utils/element_helper.dart:490]: We could use the Settings key for fetching the values? 

```dart
FlutterDriver.instance.settings.getSetting(FlutterSettings.flutterScrollDelta)
```

File: server/lib/src/utils/element_helper.dart [hunk 2]
```diff
@@ -492,8 +486,10 @@ class ElementHelper {
...
+    maxScrolls ??= FlutterDriver.instance.settings
+        .getSetting('flutterScrollMaxIteration');
```
Comment (sudharsan-selvaraj) [inline server/lib/src/utils/element_helper.dart:492]: same as above

File: server/lib/src/utils/element_helper.dart (hunk 1 above)
Comment (rahul-kumar-sdet) [reply server/lib/src/utils/element_helper.dart]: Thanks or prompt review @sudharsan-selvaraj,I have incorporated the review comments

File: server/lib/src/utils/element_helper.dart (hunk 2 above)
Comment (rahul-kumar-sdet) [reply server/lib/src/utils/element_helper.dart]: Thanks or prompt review @sudharsan-selvaraj Incorporated the review comments

File: server/lib/src/utils/element_helper.dart [hunk 3]
```diff
@@ -17,16 +17,13 @@ import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
 
+import '../driver.dart';
+import 'flutter_settings.dart';
+
```
Comment (sudharsan-selvaraj) [inline server/lib/src/utils/element_helper.dart:22]: Update the imports as below

```dart
import 'package:appium_flutter_server/src/driver.dart';
import 'package:appium_flutter_server/src/utils/flutter_settings.dart';
```

**Review by sudharsan-selvaraj — CHANGES_REQUESTED** [review]

**Comment by sudharsan-selvaraj** [comment]
Overall changes looks good. Just update the import and the PR can be merged!

File: server/lib/src/utils/element_helper.dart (hunk 3 above)
Comment (rahul-kumar-sdet) [reply server/lib/src/utils/element_helper.dart]: Done

**Review by sudharsan-selvaraj — APPROVED** [review]

**Pull request merged**